Tests/StreamChatUITests/AssertSnapshot.swift

```diff
@@ -90,3 +91,16 @@ func AssertSnapshot(
         )
     }
 }
+
+@available(iOS 13.0, *)
+/// Snapshot of a UIViewControllerRepresentable. All variants will be tested by default. For each variant, it will take a snapshot.
+/// This uses the default view controller screen size.
+func AssertSnaphot<View: UIViewControllerRepresentable>(
+    _ view: View,
+    line: UInt = #line,
+    file: StaticString = #file,
+    function: String = #function
+) {
```

Comment:
It would be great if this overload matched the existing one so we have unified API:
```
func AssertSnapshot(
    _ vc: UIViewController,
    isEmbeddedInNavigationController: Bool = false,
    variants: [SnapshotVariant] = SnapshotVariant.all,
    screenSize: CGSize = defaultScreenSize,
    suffix: String? = nil,
    record: Bool = false,
    line: UInt = #line,
    file: StaticString = #file,
    function: String = #function
) {
```

Reply:
Adjusted.